FirebaseCombineSwift.podspec

```diff
@@ -0,0 +1,74 @@
+Pod::Spec.new do |s|
+  s.name             = 'FirebaseCombineSwift'
+  s.version          = '7.6.0'
+  s.summary          = 'Swift extensions with Combine support for Firebase'
+
+  s.description      = <<-DESC
+Combine Publishers for Firebase.
+                       DESC
+
+  s.homepage         = 'https://firebase.google.com'
+  s.license          = { :type => 'Apache', :file => 'LICENSE' }
+  s.authors          = 'Google, Inc.'
+
+  s.source           = {
+    :git => 'https://github.com/firebase/firebase-ios-sdk.git',
+    :tag => 'CocoaPods-' + s.version.to_s
+  }
+
+  s.social_media_url = 'https://twitter.com/Firebase'
+  s.swift_version         = '5.0'
+  s.ios.deployment_target = '13.0'
+  s.osx.deployment_target = '13.0'
```

Comment:
This is incorrect. Please update to use ruby variables like https://github.com/firebase/firebase-ios-sdk/blob/master/FirebaseMessaging.podspec#L23